lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -112,7 +112,7 @@ def run_kubectl(*args)
     end
 
     def log_status
-      STDOUT.puts "[KUBESTATUS] #{JSON.dump(status_data)}"
+      STDOUT.puts "[KUBESTATUS #{@context}] #{JSON.dump(status_data)}"
```

Comment:
I'm in favor of `[KUBESTATUS][context][namespace]`. Great suggestion!

> Any reason the line as a whole just isn't a single JSON object containing context, namespace, etc?

For me the idea is that the JSON object is the status of the resource and `[KUBESTATUS]` is the annotation